content/docs/1.6.0/advanced-resources/deploy/node-selector.md

```diff
@@ -48,7 +48,7 @@ You need to set node selector for both types of components. See more details bel
 
 > **Warning**:
 > * Since all Longhorn components will be restarted, the Longhorn system is unavailable temporarily.
-> * Make sure all Longhorn volumes are `detached`. If there are running Longhorn volumes in the system, this means the Longhorn system cannot restart its components and the request will be rejected.
+> * Ensure that all Longhorn volumes are detached. When volumes are in use, only Instance Manager is restarted conditionally so you might be required to configure this setting again when all volumes are finally detached.
```

Comment:
Because the behavior of restarting user-deployed or system-managed components should be the same for some dangerous settings, I would recommend adding the following note next the `Danger Zone` section, so we don't repeat the same explanation.

> For * settings, when all volumes are detached, the setting will be applied immediately, and user-deployed and system-managed components will get restarted. Otherwise, the setting will not be applied until all volumes are detached during the next setting sync-up time. So, you might be required to configure this setting again to apply it immediately when all volumes are finally detached.

Reply:
@mantissahz Please help update the PR, and we should make sure users will understand when the setting will be applied or not.

Reply:
@innobead,
We only restart Longhorn system-managed components now for users who deploy Longhorn using GitOps tools.
https://github.com/longhorn/longhorn-manager/commit/a02bea80295b67691806226ae03a8d82f0a9eda6

Reply:
Added the recommended note into the `Danger Zone Setting Configuration` section in `Important note.`